dotCMS/build.gradle

```diff
@@ -148,7 +148,6 @@ dependencies {
     felix(group: 'org.apache.tika', name: 'tika-bundle', version: '1.17') {
         transitive = false
     }
-    felix group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.25'
```

Comment:
This change will cause issues, please look at this comment: https://github.com/dotCMS/core/issues/15663#issuecomment-447967132

Reply:
And comment https://github.com/dotCMS/core/issues/15663#issuecomment-448348495